scripts/cicd/pre-upload.sh

```diff
@@ -2,7 +2,7 @@
 TESTENV=$1
 
 pip install virtualenv
-virtualenv -p python3 env
+virtualenv --no-download -p python3 env
```

Comment:
"python3" is perhaps rather vague: can we commit to (at least) 3.6, say?

Reply:
this script is managed by devops team and that change was a fix made today by @say5 . 
My understanding is: the `virtualenv` mentioned in this change is responsible solely for uploading the artifact from Drone to Github, and as long as it works, we don't really worry. Perhaps @say5  has something to add  though :)